packages/terms-of-service/src/class-terms-of-service.php

```diff
@@ -60,15 +60,11 @@ public function reject() {
 	 * @return bool
 	 */
 	public function has_agreed() {
-		if ( ! $this->get_raw_has_agreed() ) {
-			return false;
-		}
-
 		if ( $this->is_development_mode() ) {
 			return false;
 		}
 
-		return $this->is_active();
+		return $this->get_raw_has_agreed() || $this->is_active();
```

Comment:
`get_raw_has_agreed` is a simpler call than `is_active`, so I agree with this ordering. It'll be more performant. 

The only edge case is, upon disconnection, does that imply the customer is revoking their "use" of Jetpack or do we state that any use of Jetpack following that.

In either event, that's a separate issue that can be discussed/implemented in a later PR.

Reply:
The intended behavior is the TOS agreement will last the lifespan of the site, so this is staying true to that.